### ISSUE TITLE
Expendable functionality for CLI 

### ISSUE THREAD

**Issue by lukaswolfrum** [amosproj/amos2022ss07-nft-playbook#8]
## User story
1. As a power user
2. I want / need to be able to easily reuse the program
3. So that it can be included in many different use cases/programs

## Acceptance criteria
* easily includable in different programs
* easily expandable functionality of CLI
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
